docs/TuxStores.md

```diff
@@ -0,0 +1,172 @@
+A store instance is able to invoke the `addChangeListener` method, which will invoke the provided callback function when the specified event type string is emitted. The change event can be either the default event string `CHANGE` or one provided by the user.
+
+```
+    store.addChangeListener(callback, CHANGE_EVENT);
+```
+
+#####<a id="addChangeListener-callback"></a>Parameter - `callback` - type: FUNCTION - required [#](#addChangeListener-callback)
+The callback function that will be called when the change event type string that is being listened for with this invocation of addChangeListener is broadcast.
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+    store.addChangeListener(listenerCallback);
+```
+
+#####<a id="addChangeListener-CHANGE_EVENT"></a>Parameter - `CHANGE_EVENT` - type: STRING - optional [#](#addChangeListener-CHANGE_EVENT)
+Change event type string that will be broadcast for any listeners that are listening for that event type. If no `CHANGE_EVENT` is specified the default value of `CHANGE` will be used.
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+    var newChangeEvent = 'NEW_CHANGE_EVENT';
+    store.addChangeListener(listenerCallback, newChangeEvent);
+```
+
+
+###<a id="store-removeChangeListener"></a>Method `store.removeChangeListener` - type: FUNCTION [#](#store-removeChangeListener)
+A store instance is able to invoke the `removeChangeListener` method, which will remove specified event type string and callback combination from the store. When removing a change listener ensure that the callback and CHANGE_EVENT input parameters are the same values that were submitted when adding the change listener. The change event can be either the default event string `CHANGE` or one provided by the user.
+
+```
+    store.removeChangeListener(callback, CHANGE_EVENT);
+```
+
+#####<a id="removeChangeListener-callback"></a>Parameter - `callback` - type: FUNCTION - required [#](#removeChangeListener-callback)
+The callback function that will be removed from the stores set of listeners when the `removeChangeListener` is invoked.
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+    store.removeChangeListener(listenerCallback);
+```
+
+#####<a id="removeChangeListener-CHANGE_EVENT"></a>Parameter - `CHANGE_EVENT` - type: STRING - optional [#](#removeChangeListener-CHANGE_EVENT)
+Change event type string that will be used to remove the listener from the store. If no `CHANGE_EVENT` is specified the default value of `CHANGE` will be used.
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+    var newChangeEvent = 'NEW_CHANGE_EVENT';
+    store.removeChangeListener(listenerCallback, newChangeEvent);
```

Comment:
Again add the listener before removing it and consider adding a newline after 132.